itou/users/management/commands/new_users_to_brevo.py

```diff
@@ -0,0 +1,203 @@
+import enum
+import logging
+import time
+
+import httpx
+from allauth.account.models import EmailAddress
+from django.conf import settings
+from django.db.models import Exists, OuterRef, Q
+from sentry_sdk.crons import monitor
+
+from itou.companies.enums import SIAE_WITH_CONVENTION_KINDS
+from itou.companies.models import CompanyMembership
+from itou.prescribers.enums import PrescriberOrganizationKind
+from itou.prescribers.models import PrescriberMembership
+from itou.users.enums import IdentityProvider, UserKind
+from itou.users.models import User
+from itou.utils.command import BaseCommand
+from itou.utils.iterators import chunks
+
+
+logger = logging.getLogger(__name__)
+
+# https://app.brevo.com/contact/list-listing
+BREVO_LIST_ID = 31
+BREVO_API_URL = "https://api.brevo.com/v3"
+
+
+class UserCategory(enum.Enum):
+    PRECSRIBER_FT = "prescripteur FT"
+    PRESCRIBER = "prescripteur habilité"
+    ORIENTEUR = "orienteur"
+    EMPLOYEUR = "employeur"
+
+
+class BrevoClient:
+    IMPORT_BATCH_SIZE = 1000
+    DELETE_BATCH_SIZE = 150  # see https://developers.brevo.com/reference/removecontactfromlist
+    EXPORT_BATCH_SIZE = 500  # see https://developers.brevo.com/reference/getcontactsfromlist
+
+    def __init__(self):
+        self.client = httpx.Client(
+            headers={
+                "accept": "application/json",
+                "api-key": settings.BREVO_API_KEY,
+            }
+        )
+
+    def _import_contacts(self, users_data, category):
+        data = [
+            {
+                "email": user["email"],
+                "attributes": {
+                    "prenom": user["first_name"].title(),
+                    "nom": user["last_name"].upper(),
+                    "date_inscription": user["date_joined"].strftime("%Y-%m-%d"),
+                    "type": category.value,
+                },
+            }
+            for user in users_data
+        ]
+
+        response = self.client.post(
+            f"{BREVO_API_URL}/contacts/import",
+            headers={"Content-Type": "application/json"},
+            json={
+                "listIds": [BREVO_LIST_ID],
+                "emailBlacklist": False,
+                "smsBlacklist": False,
+                "updateExistingContacts": True,
+                "emptyContactsAttributes": False,
+                "jsonBody": data,
+            },
+        )
+        if response.status_code != 202:
+            logger.error("BrevoAPI: Some emails were not imported", exc_info=True)
```

Comment:
Nit : ?

```suggestion
            logger.error("Brevo API: Some emails were not imported", exc_info=True)
```

+ autre occurrence l.113